test/new-e2e/pkg/components/remotehost.go

```diff
@@ -143,6 +144,10 @@ func (h *RemoteHost) RemoveAll(path string) error {
 	return clients.RemoveAll(h.client, path)
 }
 
+func (h *RemoteHost) DialRemotePort(port uint16) (net.Conn, error) {
```

Comment:
I'm on the fence. I could just expose `h.client.Dial` as it is, so you can dial anything from the host, but I'm not sure if that's desirable.